content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Using HPE GreenLake Console's API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
+---
+## Secured yet Agile
+
+One major principle of Application Programming Interface (API) for the HPE Data Services Cloud Console (DSCC) is secured, but also flexible for consumption of multiple applications or tools that use this API to extend their function using the DSCC. One of many features for the delegated authorization below includes the ability to provide access to the API resources inside the DSCC (such as volumes, arrays, etc.) without embedding the user's credential (username, and password) in the API communication stream.
+
+### Authentication Process to Obtain the Access Token
+
+The DSCC public API relies on an OAuth 2.0 third party authorization framework on behalf of the resource owner (HPE GreenLake Console's user) for security. The user starts by logging and authenticating into HPE GreenLake Console, which is authenticated by the Identity Provider (validated through username, password, or MFA). Using the API Gateway menu in GreenLake Console, customer registers their client application to obtain the OAuth 2.0 API Client Credentials (Client ID and Client Secret). This association allows user to obtain the access-token from the menu, and user can then use the access-token into any REST API call inside the token bearer field. This action allows any client application or script to perform any HTTPS method to 
```

Comment:
"third party" should be "third-party"
Please spell out "MFA" on its first use. You can then put "(MFA)" in parentheses afterwards.
Change "...menu in GreenLake Console" to "...menu in HPE GreenLake Console"
Add "a" prior to "customer registers"
Add "a" after "allows"/prior to "user" (allows a user)
Add "the" after "and"/prior to "user can then" (...the menu, and the user can then"
Not sure that the word "into" is the right word (just prior to "any REST API call" - maybe "for"?
At the very end, some words appear to be missing, as it just ends with "to perform any HTTPS method to" and there's no period.